templates/default/src/html/partials/base/base-ad-script.hbs

```diff
@@ -13,7 +13,7 @@
 	var slots = {};
 	googletag.cmd.push(function () {
 		var pgTargeting = {};
-		var section = '{{@root.meta.sectionUrl}}'.replace(/http[s]*?:\/\//, '');
+		var section = '{{@root.meta.sectionUrl}}'.replace(/http[s]*?:\/\/www\./, '');
```

Comment:
In the edge case where `www` isn't present, this will fail to work properly. Inserting the `www.` in a group will solve that:
```.replace(/http[s]*?:\/\/(www\.)?/, '')```

Reply:
Good call, thanks